src/views/Tenant/TenantView.js

```diff
@@ -165,9 +165,9 @@ const Tenant = () => {
       comp: <div />,
     },
     {
-      key: "unit",
-      label: "Unit",
-      value: property.unit,
+      key: "unitNum",
+      label: "Total Units",
+      value: tenant.lease && tenant.lease.unitNum,
```

Comment:
So currently, the Tenant model has a one to many relationship called `leases` rather than lease.  I was talking with @NickSchimek about it and we may change it to be a one to one called `lease`, in which case this would work.

However, with the current model, this line would need to be:
```suggestion
      value: tenant.leases[0] && tenant.leases[0].unitNum,
```

*(pending proper lease serialization on the backend &mdash; see other comment)

Also, line 116 would need to be changed to:
```javascript
const propertyUrl = `/api/properties/${tenant.leases[0].propertyID}`;
```
Otherwise, `undefined` gets passed in as the property ID.

Reply:
Thanks for reviewing @aedwardg Here are some thoughts.

I think irregardless of how we have it setup in the backend. We should probably be only sending one lease to the frontend, unless they have a need for all leases. If the frontend uses `leases[0] then I don't think we could guarantee that the current or relevant lease will always be at index 0. So I would suggest that the frontend does not need to know how the relationship exists, but rather only needs to know that we are going to send them a json object with a lease nested inside a tenant. so the way @Chris-Boe has it should be fine.

For example. The backend will send this JSON object and will guarantee that the nested lease object contains one lease and is the relevant/current lease. So even though we may have a many-to-one relationship in the back we will be filtering out all the leases except for the relevant one and serialize that one inside the tenant object.
```
{ 
    tenant: {
        name: 'name'
        ...
        lease: {
            unitNum: 'd405'
            ...
        }
    }
}
```

Reply:
Yep, good call @NickSchimek!  Looking at it again, I think you're right that the backend should handle this and just send the one lease up to the frontend.  I'll remove these suggested changes and open up a new backend PR for that.

Reply:
Thanks @aedwardg and @NickSchimek !